TelCo.ColorCoder/ColorCode.cs

```diff
@@ -0,0 +1,39 @@
+using System;
```

Comment:
there are duplicate files colorcode.cs etc..

Reply:
I was created duplicate file outside the project, due to lack of knowledge of GitHub. I have cleared the unwanted file from the solution.